docs-src/references/commands.md

```diff
@@ -50,27 +50,35 @@ This Command is triggered by a call to spawn a [Child Workflow Execution](/conce
 - Awaitable: Yes, a Workflow Execution can await on the action resulting from this Command.
 - Corresponding Event: [ChildWorkflowExecutionStarted](/references/events/#childworkflowexecutionstarted)
 
+By default, OSS users cannot have more than 50,000 pending Child Workflows.
```

Comment:
We should mention that operator of temporal cluster could adjust this limit via dynamic config, and list the config key that controls this limit.

Reply:
Yes, and you can find the key names here https://www.notion.so/temporalio/Workflow-Size-Limits-877a2b2d4f74457d9533f0f976ca442a in the table under "Dynamic Config Key". Don't mention proto field name, though.